include/quicly/cc.h

```diff
@@ -0,0 +1,38 @@
+/* Interface definition for quicly's congestion controller.
+ */
+
+#ifndef quicly_cc_h
+#define quicly_cc_h
+
+#include "quicly/constants.h"
```

Comment:
It might be a good idea to move this include below the system header files, so that we do not mess with the system definitions.